### PR TITLE
DYN-5530 library slider tooltip

### PR DESCRIPTION
**Pull request by filipeotero**
### Purpose

It's included in this PR a tooltip to show the value of the library's zoom slider. Also, the fix for the help icon.

sliderTooltip

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### FYIs

@QilongTang @avidit

### CODE REVIEW

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml [hunk 1]
```diff
@@ -1273,16 +1273,17 @@
                                         <RowDefinition Height="Auto"/>
                                     </Grid.RowDefinitions>
                                     <StackPanel Margin="0,10,0,5" Grid.Row="0" Orientation="Horizontal">
-                                        <TextBlock  FontSize="12" Text="Library"></TextBlock>
+                                        <TextBlock  FontSize="12" VerticalAlignment="Center" Text="Library"></TextBlock>
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml:1276]: I did not notice this before, it seems the text `Library` is missing localization this way

File: src/LibraryViewExtensionWebView2/LibraryViewController.cs [hunk 2]
```diff
@@ -88,7 +88,8 @@ internal LibraryViewController(Window dynamoView, ICommandExecutive commandExecu
 
         private void Browser_ZoomFactorChanged(object sender, EventArgs e)
         {
-            dynamoViewModel.Model.PreferenceSettings.LibraryZoomScale = ((float)browser.ZoomFactor);
+            //Multiplies by 100 so the value can be saved as a percentage
+            dynamoViewModel.Model.PreferenceSettings.LibraryZoomScale = (float)browser.ZoomFactor * 100;
```
Comment (QilongTang) [inline src/LibraryViewExtensionWebView2/LibraryViewController.cs:92]: If we use this approach, the number should be int instead of float right?

Comment (filipeotero) [reply src/LibraryViewExtensionWebView2/LibraryViewController.cs]: Yes, changed the variable to int

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml (hunk 1 above)
Comment (filipeotero) [reply src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml]: Thanks for noticing, added a resource for that.

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
@filipeotero Please take a look at the regression [TestImportCopySettings](https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/8565/testReport/junit/Dynamo.Tests.Configuration/PreferenceSettingsTests/TestImportCopySettings/)

**Comment by zeusongit** [comment]
Hey @filipeotero, not to extend the scope of this PR too much, if it is not a lot of work (in which case we can create a separate task) can you add the current zoom level indicator, like we have for ML autocomplete confidence level slider?
Screenshot 2023-02-23 at 6 40 21 PM

**Comment by filipeotero** [comment]
> Hey @filipeotero, not to extend the scope of this PR too much, if it is not a lot of work (in which case we can create a separate task) can you add the current zoom level indicator, like we have for ML autocomplete confidence level slider? Was there a reason it was added as a tooltip and not a fixed element? Screenshot 2023-02-23 at 6 40 21 PM

I used the native Autotooltip from the slider. I will adapt it as it is in the ML autocomplete since we discussed in the Jira task that we want it to have the same behavior. Thank you for noticing that!

**Comment by filipeotero** [comment]
fixedTooltipSliderNew

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs [hunk 3]
```diff
@@ -530,5 +530,20 @@ private void displayConfidenceLevel()
                 this.lblConfidenceLevel.Margin = new Thickness(left, -15, 0, 0);
             }
         }
+
+        private void zoomScaleLevel_ValueChanged(object sender, RoutedPropertyChangedEventArgs<double> e)
+        {
+            Slider slider = (Slider)sender;
+
+            //Since the percentage goes from 10 to 300, the value is decremented by 10 to standardize. 
+            double percentage = slider.Value - 10;
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs:539]: Now that it starts with 10%, thats why we minus 10 as a starting point?

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs [hunk 4]
```diff
@@ -530,5 +530,20 @@ private void displayConfidenceLevel()
...
+            //This is the relation between the margin in pixels and the value of the percentage
+            double marginValue = (79 * percentage / 29) - 480;
+            if (lblZoomScalingValue != null)
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs:542]: When would this value be null, do you have an example?

**Review by QilongTang — COMMENTED** [review]
LGTM with two comments

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs (hunk 3 above)
Comment (filipeotero) [reply src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs:539]: Yes, to facilitate the calculation.

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs (hunk 4 above)
Comment (filipeotero) [reply src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs:542]: When the component is initialized, this method is called, but the value of the zoom scale is null.

**Pull request merged**